nav/nav_stack/config/local_costmap_params.yaml

```diff
@@ -5,11 +5,44 @@ local_costmap:
   publish_frequency: 2.0
   static_map: false
   rolling_window: true
-  width: 8.0
-  height: 8.0
+  width: 40.0 # cost map size needs to be large enough to encompass the point cloud range
+  height: 40.0 # This size is arbitary for now so it may need to be adjusted
```

Comment:
does this computationally work? I feel like this is very large, and should be tested on crappier comps

Reply:
That's fair enough. I can keep playing around with the size to find the minimum size to make it work.

Reply:
20x20:
![20x20](https://user-images.githubusercontent.com/74154406/150649570-c07174e1-1aea-4500-bccb-c31b8b0bf1a6.png)

25x25:
![25x25](https://user-images.githubusercontent.com/74154406/150649582-e2ea5871-8d9a-4d41-ba96-b44603267fc7.png)

30x30:
![30x30](https://user-images.githubusercontent.com/74154406/150649586-2e3c9549-49c8-4e09-a57e-17eab69c32ea.png)

35x35:
![35x35](https://user-images.githubusercontent.com/74154406/150649596-4031dea8-af49-4de7-b5b5-872e5fa4b850.png)

Reply:
It doesn't seem like there's much difference between 30x30 and 35x25 so I think the minimum size can be set to 30x30 for now.

Reply:
Resolved by setting the local cost map size to the minimum required map size of 30x30